main/modes/music/tunernome/tunernome.c

```diff
@@ -277,6 +277,8 @@ static int TUNER_FLAT_THRES_X;
 static int TUNER_SHARP_THRES_X;
 static int TUNER_THRES_Y;
 
+// TODO: We can't use the mic and the speaker anymore, so I don't really know what to do with this anyway.
```

Comment:
We can still use them individually and should be able to swap between modes at runtime. The tuner uses the mic and the metronome uses the speaker, so that should be feasible.

Reply:
Ah yeah, true. I think the other thing that I need to do here is either turn the metronome clicks into an actual MIDI track and have the metronome just set the tempo and be done with it. Or just add support for playing metronome clicks in the MIDI player directly, I think that might also be a common thing...